src/commands/bans/banned.ts

```diff
@@ -0,0 +1,16 @@
+import {Command} from "@ubccpsc310/bot-base";
+import {Client, Message} from "discord.js";
+import {getBannedWord} from "../../controllers/BanController";
+
+const banned: Command = {
+    name: "banned",
+    description: "Lists the banned word (if any)",
+    usage: "banned",
+    procedure: async (client: Client, message: Message) => {
+        const bannedWord = await getBannedWord();
+        const reply = bannedWord ? `The current banned word is: ||${bannedWord}||` : "No word is banned at the moment";
```

Comment:
This is just a suggestion:
if bannedWord contains a vowel
   replace middle-most vowel with `*`

Just funnier I think

Reply:
I like that idea!